### PR TITLE
10.5.0 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/10/10.5.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+**Release date: November 28, 2023**
+
+{{% button color="info" href="https://marketplace.mendix.com/link/studiopro/10.5.0" text="Go to Marketplace" title="Download version 10.5.0 from the Marketplace" %}}
+
+### New Features
+
+#### XPath Builder
+
+We have added a new beta visual editor for writing XPath constraints. The XPath builder is an easy-to-use and intuitive tool for quickly developing the most common database constraint use cases without having to learn and write XPaths.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/studio-pro/10/10.5.md]: I think that it isn't a beta visual editor. It is a visual editor which happens to be beta.
Also adding a link to XPath where the documentation will be.
```suggestion
We have added a new visual editor for writing [XPath](/refguide/xpath/) constraints. The XPath builder is an easy-to-use and intuitive tool for quickly developing the most common database constraint use cases without having to learn and write XPaths. The XPath builder is currently in beta.
```

File: content/en/docs/releasenotes/studio-pro/10/10.5.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+You can open the new visual builder by clicking **Builder (beta)** in the **XPath constraint** dialog box (and then switch back to the XPath expression editor by clicking **XPath expression**). The XPath Builder can also be made the default editor via **Edit** > **Preferences** > [New features](/refguide/preferences-dialog/#new-features).
+
+Some advanced XPath features are not yet supported by the XPath Builder. For those cases, a message is shown stating that the XPath constraint contains advanced features and is not yet supported by the XPath Builder.
+
+Along with the introduction of the new visual XPath Builder, we have also taken the opportunity to unify the **Database** and **XPath** data sources into the **Database** data source.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/studio-pro/10/10.5.md]:
```suggestion
Along with the introduction of the new visual XPath Builder, we have also taken the opportunity to combine the **Database** and **XPath** data sources under the **Database** data source.
```

**Review by MarkvanMents — COMMENTED** [review]
@Adam-Dupaski 
A couple of suggestions about XPath builder, but feel free to ignore if you don't think they add anything.

File: content/en/docs/releasenotes/studio-pro/10/10.5.md (hunk 1 above)
Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/10/10.5.md]: Will `Builder` be capitalized in the UI @MarkvanMents ?

**Comment by Adam-Dupaski** [comment]
Will `XPath Builder` be capitalized in the UI @MarkvanMents ?

**Comment by MarkvanMents** [comment]
> Will `XPath Builder` be capitalized in the UI @MarkvanMents ?

Builder is capitalized in the two places where it is currently used.
image
image

**Comment by Adam-Dupaski** [comment]
> > Will `XPath Builder` be capitalized in the UI @MarkvanMents ?
> 
> Builder is capitalized in the two places where it is currently used. image image

Okay, will use "XPath **Builder**"

**Pull request merged**